Styling adjustments

SLeitgeb opened the pull request:
Some opinionated changes.

Let's make the map background a sea blue colour instead of white.

Remove the opacity from fill colours and make them a lighter shade to match the colour they would be if the original opacity were applied.

The entry for 0 students (grey colour) was missing in the legend.

SLeitgeb committed:
Simplify geometry using pixel area

Fixes Antarctica artefacts in #10. Using the `--spherical-area`
toposimplify parameter correctly preserves area, but breaks geometry
near the south pole. Using `--planar-area` solves the issue, but applies
simplification unevenly with latitude.

Projecting the data to the same CRS as the final map and then
simplifying might be the better choice down the road, since the usage of
`--planar-area` parameter will be actually correct.

SLeitgeb committed:
Apply nicer formatting

SLeitgeb committed:
Add a helper file to fill the Egypt artefacts

Fixes #10.

SLeitgeb committed:
Remove margins from html and body

Let the map fill all of the available space.

SLeitgeb committed:
Remove opacity from the fill colors

SLeitgeb committed:
Add a missing legend entry (0 students)

DajanaSnopkova commented on style.css:
```diff
@@ -1,18 +1,23 @@
+    background-color: #C6ECFF;
+    top: 0;
+    bottom: 0;
+    width: 100%;
```
I personally do not like the blue background. It also does not look so good on the final website. I tested it [here](https://webcentrum.muni.cz/antropologie-testovaci/en/testovani-narodnosti-studentu/int-zkusebni):  

Also, I am not sure, if the width 99% and width 100% is making this difference with the occurring horizontal scrollbar when put in iframe.

SLeitgeb committed:
Revert the map background color to white

Use '#FAFAFA' to match the background-color value of the target page
background.

DajanaSnopkova commented on map.js:
```diff
@@ -12,11 +12,11 @@ function getColor(d) {
+         d >  30 ? '#19773A' :
+         d >  10 ? '#199746' :
+         d >   0 ? '#19B651' :
+                   '#CBCBCB';
```
I suggest #D9D9D9.

DajanaSnopkova commented on map.js:
```diff
@@ -91,12 +96,12 @@ function stateStyle(feature) {
     // choropleth map style
     fillColor: getColor(feature.properties[studentsCountAttr]),
-    color: 'white',
+    color: '#aaa',
```
I suggest the border color be the same as the background #fafafa.

DajanaSnopkova commented on map.js:
```diff
@@ -91,12 +96,12 @@ function stateStyle(feature) {
     /* //old variant - no choropleth map
       fillColor: feature.properties[studentsCountAttr] > 0 ? '#00AF3F' : '#C6C6C6',
       color: feature.properties[studentsCountAttr] > 0 ? '#007D2C' : '#9E9E9E', */
     weight: 0.8,
```
Maybe thicker.. 1.2?

SLeitgeb committed:
Draw only land boundaries

Since the boundary colour is the same as the map background, drawing the
boundaries as polygon stroke masks coastlines and islands.

Since we use TopoJSON, drawing land boundary mesh is trivial.
